src/main/java/game/Logger.java

```diff
@@ -0,0 +1,67 @@
+package game;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.logging.Level;
+
+/**
+ * This class will log every action.
+ * @author Kibo
+ *
+ */
+public class Logger {
+	/**
+	 * The log file.
+	 */
+	private final File file;
+	/**
+	 * The file output stream.
+	 */
+	private FileOutputStream fos;
+	/**
+	 * The Date format.
+	 */
+	private final SimpleDateFormat sdf;
+	/**
+	 * This Game.
+	 */
+	private final Game game;
+	
+	/**
+	 * Constructor of Logger.
+	 *@param game - this game
+	 */
+	public Logger(final Game game) {
+		file = new File("src/main/resources/log.txt");
```

Comment:
voor nu, gooi dit bestand gewoon in dezelfde locatie als de applicatie, we zetten het later wel op een betere locatie